src/components/ListLayout.jsx

```diff
@@ -1,15 +1,20 @@
 import React, { useEffect, useState, useRef } from 'react';
-import { updateDoc, doc } from 'firebase/firestore';
+import { updateDoc, doc, deleteDoc } from 'firebase/firestore';
 
 import { db } from '../lib/firebase';
 import { ImCross } from 'react-icons/im';
+import { RiDeleteBin6Fill } from 'react-icons/ri';
 import { calculateEstimate } from '@the-collab-lab/shopping-list-utils';
 
 // UPDATED this function to accommodate changes to multiple values on an item object
 const setUpdateToDb = async (collection, itemId, dataToUpdate) => {
   const itemRef = doc(db, collection, itemId);
   await updateDoc(itemRef, dataToUpdate);
```

Comment:
`setUpdateToDb` and `deleteItemFromDb` are good candidates to move into a lib or util file (they could be added to the existing `firebase.js`). For the purposes of this app, I think you will only be updating/deleting from here, but these are great reusable functions that could be used in other components as the app grows.